qemu/tests/x86_cpu_flag_nonstop_tsc.py

```diff
@@ -0,0 +1,48 @@
+import logging
+
+from virttest import env_process
+from virttest import error_context
+
+from provider.cpu_utils import check_cpu_flags
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Test cpu flag nonstop_tsc.
+    1) Check if current flags are in the supported lists on host, if no, cancel test
+    2) Otherwise, boot guest with the cpu flag 'invtsc'
+    3) Check cpu flags inside guest(only for linux guest and not for RHEL6)
+    4) Check tsc inside guest(only for linux guest)
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    flag = params["flags"]
+    check_host_flags = params.get_boolean("check_host_flags")
+    if check_host_flags:
+        check_cpu_flags(params, flag, test)
+
+    params["start_vm"] = "yes"
+    vm_name = params['main_vm']
+    env_process.preprocess_vm(test, params, env, vm_name)
+
+    vm = env.get_vm(vm_name)
+    error_context.context("Try to log into guest", logging.info)
+    session = vm.wait_for_login()
+    if params["os_type"] == "linux":
+        if params['os_variant'] != 'rhel6':
+            check_cpu_flags(params, flag, test, session)
+        check_clock = params["check_clock"]
+        check_clock_out = session.cmd_output(check_clock)
```

Comment:
suggest to use session.cmd_status.

Reply:
> suggest to use session.cmd_status.

@huangyum thanks and update.

Host_RHEL.m8.u4.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.9.x86_64.io-github-autotest-qemu.x86_cpu_flags.others.nonstop_tsc: PASS (118.34 s)